hasher-matcher-actioner/webapp/src/settings/ActionSettingsTab.jsx

```diff
@@ -0,0 +1,281 @@
+            variant="primary"
+            onClick={() => {
+              setEditing(true);
+            }}>
+            Edit
+          </Button>
+        </Card.Footer>
+      </div>
+
+      <div hidden={!editing}>
+        <Card.Header>
+          <Form>
+            <Form.Group>
+              <Form.Control as="select" size="lg">
+                <option>WebhookActioner</option>
+              </Form.Control>
+            </Form.Group>
+          </Form>
+        </Card.Header>
+        <Card.Body>
+          <Form>
+            <Form.Group>
+              <Form.Label>URL</Form.Label>
+              <Form.Text className="text-muted">
+                {actionerDetails.args.url.description}
+              </Form.Text>
+              <Form.Control
+                type="url"
+                value={tempURL}
+                onChange={e => {
+                  setTempURL(e.target.value);
+                }}
+              />
+              <Form.Label>Webhook Type</Form.Label>
+              <Form.Text className="text-muted">
+                {actionerDetails.args.webhookType.description}
+              </Form.Text>
+              <Form.Control
+                as="select"
+                value={tempWebhookType}
+                onChange={e => {
+                  setTempWebhookType(e.target.value);
+                }}>
+                <option>POST</option>
+                <option>GET</option>
+                <option>PUT</option>
+                <option>DELETE</option>
+              </Form.Control>
+
+              <Form.Label>Headers</Form.Label>
+              <Form.Text className="text-muted">
+                {actionerDetails.args.headers.description}
+              </Form.Text>
+              <Form.Control
+                type="text"
+                value={tempHeaders}
+                onChange={e => {
+                  setTempHeaders(e.target.value);
+                }}
+              />
+            </Form.Group>
+          </Form>
+        </Card.Body>
+        <Card.Footer>
+          <Button
+            variant="primary"
+            onClick={() => {
+              setURL(tempURL);
+              setWebhookType(tempWebhookType);
+              setHeaders(tempHeaders);
+              setEditing(false);
+            }}>
+            Save
+          </Button>
+          <Button
+            variant="danger"
+            onClick={() => {
+              setTempURL(savedURL);
+              setTempWebhookType(savedWebhookType);
+              setTempHeaders(savedHeaders);
+              setEditing(false);
+            }}>
+            Cancel
+          </Button>
+        </Card.Footer>
+      </div>
+    </Card>
+  );
+}
+
+const ActionRenderers = {
+  WebhookActioner: WebhookActionerRender,
+};
+
+function ActionPerformerRow({name, type, params}) {
+  const [editing, setEditing] = useState(false);
+  const [savedName, setName] = useState(name);
+  const [tempName, setTempName] = useState(name);
+  return (
+    <tr>
```

Comment:
I'm guessing if you run with developer tools open, you should be seeing an issue regarding "key" not being defined. You need that on this <tr>, I think. It needs to be set to something unique. See https://reactjs.org/docs/lists-and-keys.html for background / context.

Reply:
Hmm. I'm not seeing that error. Might be hidden? or might be because its rows instead of list items